src/extensions/surveys.tsx

```diff
@@ -99,6 +104,7 @@ export const callSurveys = (posthog: PostHog, forceReload: boolean = false) => {
                 }
 
                 if (!localStorage.getItem(getSurveySeenKey(survey))) {
+                    visibleSurveys.add(survey.id)
```

Comment:
I think we need to remove the survey.id when it's not visible as well?

I'd expect this set to always have one element. For cases when for example a survey is shown -> responded -> needs to show up again because config / iteration / whatever -> currently it wouldn't show up again if I'm reading things correctly.